ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -36,6 +36,36 @@ group_repo_post() {
     :
 }
 
+distro_custom() {
+    dnf -y $dnf_repo_args install python3
```

Comment:
(lint) Double quote to prevent globbing and word splitting. [SC2086]

Reply:
Quoting the string causes a problem:
```
[2021-03-10T08:16:16.164Z] + dnf -y '--disablerepo=* --enablerepo=repo.dc.hpdd.intel.com_repository_*' install python3
[2021-03-10T08:16:16.164Z] No repository match: --enablerepo=repo.dc.hpdd.intel.com_repository_*
[2021-03-10T08:16:16.164Z] Error: There are no enabled repos.
```